pySDC/projects/error_estimation/README.rst

```diff
@@ -0,0 +1,75 @@
+Error Estimation in SDC
```

Comment:
Please give some pointers. Should I be more formal, less formal? More elaborate, more technical, more images,...?

Reply:
I have no complaints! I don't care for formal or no formal. In any case, this project will evolve over time anyway, so for now this is great!